packages/gatsby-source-lever/src/gatsby-node.js

```diff
@@ -3,6 +3,50 @@ const normalize = require(`./normalize`)
 
 const typePrefix = `lever__`
 
+exports.createSchemaCustomization = async ({ actions }) => {
+  const typeDefs = `
+    type lever implements Node {
+      additional: String
+      additionalPlain: String
+      applyUrl: String
+      createdAt: Date
+      description: String
+      descriptionPlain: String
+      id: ID!
+      hostedUrl: String
+      lever_id: String
+      text: String
+      categories: LeverCategories
+      lists: [LeverList]
+      internal: LeverInternal!
+    }
+
+    type LeverCategories {
+      commitment: String
+      level: String
+      location:String
+      team: string
+    }
+
+    type LeverList {
+      content: String
+      text: String
+    }
+
+    type LeverInternal {
+      content: String
+      contentDigest: String!
+      description: String
+      fieldOwners: [String]
+      ignoreType: Boolean
+      mediaType: String
+      owner: String!
+      type: String!
+    }
```

Comment:
This whole type is not needed, Gatsby handles it automatically together with `Node` interface ([docs](https://www.gatsbyjs.org/docs/schema-customization/#the-node-interface))